.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: ruby
+cache: bundler
+rvm:
+  - 2.0.0-p451
+
+branches:
+  only:
+    - master
+    - /.+-stable$/
```

Comment:
it's missing the trailing newline, which can lead to merge conflicts (and my text editor automatically dirtying the file)